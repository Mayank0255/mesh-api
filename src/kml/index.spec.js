import { getKML } from ".";

describe("getKML", () => {
  it("renders KML for appointments, LoS, nodes, and requests", () => {
    expect(getKML()).toMatchSnapshot();
  });
});
