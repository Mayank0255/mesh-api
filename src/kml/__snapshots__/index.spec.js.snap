// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getKML renders KML for appointments, LoS, nodes, and requests 1`] = `
"<?xml version=\\"1.0\\" encoding=\\"UTF-8\\"?>
<kml xmlns=\\"http://www.opengis.net/kml/2.2\\" xmlns:gx=\\"http://www.google.com/kml/ext/2.2\\" xmlns:kml=\\"http://www.opengis.net/kml/2.2\\" xmlns:atom=\\"http://www.w3.org/2005/Atom\\">
<Document>
	<name>NYC Mesh API</name>
	<NetworkLink>
    <name>Appointments</name>
    <Link>
        <href>http://localhost:9000/v1/kml/appointments</href>
    </Link>
</NetworkLink>
	<NetworkLink>
    <name>LoS</name>
    <Link>
        <href>http://localhost:9000/v1/kml/los</href>
    </Link>
</NetworkLink>
	<NetworkLink>
    <name>Nodes</name>
    <Link>
        <href>http://localhost:9000/v1/kml/nodes</href>
    </Link>
</NetworkLink>
	<NetworkLink>
    <name>Requests</name>
    <Link>
        <href>http://localhost:9000/v1/kml/requests</href>
    </Link>
</NetworkLink>
</Document>
</kml>
"
`;
